### DISCUSSION TITLE
eCAL Webinar on August 24, 2023

### DISCUSSION THREAD

**Discussion by FlorianReimold** [eclipse-ecal/ecal#1189] [Announcements]
If you are interested in Eclipse eCAL, join us in today's webinar! We will be giving an overview over eCAL and complete it with a live demo of eCAL 5.12.

The live event will happen on August 24, 2023
- 16:00 CEST (Central Europe)
- 10:00 AM EDT (North America)

The webinar is streamed via cloudcast to various platforms. You can join with this link:
https://www.crowdcast.io/e/shiftinggears-08242023/register?utm_campaign=SDV%20Shifting%20Gears&utm_medium=email&_hsmi=271263947&utm_content=271263947&utm_source=hs_email

![1692799205500](https://github.com/eclipse-ecal/ecal/assets/11774314/aa845df7-f246-4815-b566-7a8ad89ed5ab)

**Comment by FlorianReimold** [comment]
You can now watch the webcast on Youtube!

https://www.youtube.com/watch?v=8AAxlu1WbdU

**Comment by felixf4xu** [comment]
Do you have a link for the slides?

**Reply by FlorianReimold** [reply]
@felixf4xu: Sure, but Kristof is on vacation right now and not available and I don't seem to have access to them.
@hannemn: Could you upload the slides here?

**Comment by FlorianReimold** [comment]
@felixf4xu: Slides are here:
[eCAL Webinar.pptx](https://github.com/eclipse-ecal/ecal/files/12595949/eCAL.Webinar.pptx)
